integration-tests/utils/UIhelper.ts

```diff
@@ -17,7 +17,8 @@ export class UIhelper {
   }
 
   static verifyLabelAndValue(label: string, value: string) {
-    return cy.contains('div', label).contains('dd', value)
+    cy.log(`Validate Label : "${label}" should have value : "${value}"`)
+    return cy.xpath(`//div[contains(@class,"list__group") and descendant::*[text()="${label}"]]/dd//*[text()="${value}"]`).scrollIntoView().should('be.visible')
```

Comment:
what was wrong with the original way of locating? this is overly specific

Reply:
list__group class helps to look for only Label and value Components/Elements

Reply:
Have tried that, its not consistent and fails to validate some elements.

Reply:
both the locator becomes same with above regex, I can revert to previous one its not currently used anywhere in tests as of now.